headers/types/common/enums.h

```diff
@@ -3649,4 +3649,14 @@ enum monster_gender {
     GENDER_GENDERLESS = 3,
 };
 
+typedef enum wan_source_type {
```

Comment:
No typedefs for enums please. You're already using `enum wan_source_type` everywhere else anyway.

```c
enum wan_source_type {
    ...
};
```